pkg/services/publicdashboards/internal/test_helpers.go

```diff
@@ -22,3 +22,16 @@ func GetTimeRangeFromDashboard(t *testing.T, dashboardData *simplejson.Json) (st
 
 	return fromUnixMilli, toUnixMilli
 }
+
+func CreateDatasource(dsType string, uid string) struct {
```

Comment:
Is this used in multiple packages? e.g. API/Service? I'm skeptical that we should even have a test helpers directory as files with _test.go cannot be imported anyway.